lms/djangoapps/django_comment_client/utils.py

```diff
@@ -59,7 +60,8 @@ def has_forum_access(uname, course_id, rolename):
     return role.users.filter(username=uname).exists()
 
 
-def _get_discussion_modules(course):
+# pylint: disable=invalid-name
+def get_accessible_discussion_modules(course, user):
     """
     Return a list of all valid discussion modules in this course.
```

Comment:
i think the docstring needs to change now (should note that it's filtered by user access)